plugins/inputs/elasticsearch/README.md

```diff
@@ -219,7 +219,6 @@ Operating system stats, load average, cpu, mem, swap measurement names:
   - cpu_user value=0
   - cpu_idle value=99
   - cpu_usage value=0
-  - cpu_stolen value=0
```

Comment:
I think this shouldn't be changed, these OS stats come from the Elasticsearch server and not gopsutil.

Reply:
Well spotted, thanks for the careful review. Updated accordingly.